pandas/_config/config.py

```diff
@@ -820,6 +821,31 @@ def inner(x):
     return inner
 
 
+def is_pos_int(value):
+    """
+    Verifies that value is None or a positive int
+
+    Parameters
+    ----------
+    value - the value to be checked
+
+    Raises
+    ------
+    ValueError if value is not equal to None or int, or if value is not
+    positive in the case of a int type.
+
+    """
+
+    if not (value is None or isinstance(value, int)):
+        msg = "Value must be an instance of <class 'NoneType'>|<class 'int'>"
```

Comment:
I think most people won't be familiar with NoneType. Should probably say like `'value' must be a positive integer or None`.